//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acd_O_P.h
//
// Code generation for function 'acd_O_P'
//

#ifndef ACD_O_P_H
#define ACD_O_P_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acd_O_P(double P_init, double V_init, double A_init, double P_wayp,
             double V_max, double J_max, double J_min, creal_T t[14]);

#endif
// End of code generation (acd_O_P.h)
